airflow-core/pyproject.toml

```diff
@@ -89,7 +89,7 @@ dependencies = [
     "flask-caching>=2.0.0",
     # Flask-Session 0.6 add new arguments into the SqlAlchemySessionInterface constructor as well as
     # all parameters now are mandatory which make AirflowDatabaseSessionInterface incompatible with this version.
-    "flask-session>=0.4.0,<0.9",
+    "flask-session>=0.8.0,<0.9",
```

Comment:
I think we can remove the above comment as it relates to 0.6 ?

Reply:
We updated this in another PR. I will update this PR once we remove flask_session from core